version.json

```diff
@@ -5,9 +5,10 @@
     ".*"
   ],
   "cloudBuild": {
-    "setAllVariables": true,
+    "setAllVariables": false,
     "buildNumber": {
-      "enabled": true
-    }
+      "enabled": false
+    },
+    "setVersionVariables": false
```

Comment:
CloudBuild variable update was required because nerbbank thinks that build is running in GitHub action container but that's not true and it tries to access file system which is not present in CodeBuild.